rio_tiler/io/stac.py

```diff
@@ -337,22 +329,29 @@ def point(
 
     def stats(
         self,
-        assets: Union[Sequence[str], str],
         pmin: float = 2.0,
         pmax: float = 98.0,
+        hist_options: Dict = {},
+        assets: Union[Sequence[str], str] = None,
         **kwargs: Any,
     ) -> Dict:
         """Return array statistics from COGs."""
+        if not assets:
+            raise InvalidBandName("Missing 'assets'")
```

Comment:
Should `InvalidBandName` be renamed to `InvalidAsset` or similar?  Assets are not always bands.

Reply:
Yep we should have `InvalidAssetName` and `MissingAssetName` will add that in another PR